integration-tests/cra4/tests/playwright/apollo-client.test.ts

```diff
@@ -0,0 +1,10 @@
+import { expect } from '@playwright/test';
+import { test } from 'shared/fixture';
+
+test('Basic Test', async ({ page, withHar }) => {
+  await page.goto('http://localhost:3000');
+
+  await expect(page.getByText('loading')).toBeVisible();
+  await expect(page.getByText('loading')).not.toBeVisible();
```

Comment:
```suggestion
  await expect(page.getByText('loading')).not.toBeVisible({ timeout: 10000 });
```

Should this include the timeout like the other tests?

Reply:
got it. Thanks!